src/pages/schaapi.js

```diff
@@ -1,27 +1,102 @@
+                </a>
+                <a href="https://www.tudelft.nl/">
+                    <img src={tudelftLogo} title="Delft University of Technology" height={50} />
+                </a>
+            </div>
         </div>
 
-        <div className="flex justify-center align-bottom items-center mt-8">
-            <a href="http://swerl.tudelft.nl/bin/view/Main/WebHome">
-                <img src={sergLogo} title="Software Engineering Research Group" height={35} className="mr-4 mt-3" />
-            </a>
-            <a href="https://www.tudelft.nl/">
-                <img src={tudelftLogo} title="Delft University of Technology" height={50} />
-            </a>
+        <div className="mt-16">
+            <div className="content-section bg-delft-blue text-white">
+                <h2>What is a "schaapi"?</h2>
+
+                <p className="text-xl text-center">
+                    <strong>Schaapi is a tool for early detection of breaking changes based on API usages!</strong>
+                </p>
+
+                <p className="text-sm text-center">It also contains the Dutch word for sheep ("schaap") 🐑</p>
+            </div>
+            <div className="content-section bg-red-lighter">
+                <h2>But why?</h2>
+                <p>
+                    <strong>Library developers</strong> are often <strong>unaware</strong> of how their library is used
+                    exactly <strong>in practice</strong>. When a library developer changes the internals of a library,
+                    this may unintentionally affect or even break the working of the library users' code. While it is
+                    possible to detect when a syntactic breaking change occurs, it is not as easy to detect semantic
+                    breaking changes, where the <strong>implicit contract of a functionality</strong> changes, sometimes
+                    unbeknownst to the library developer. Because library users rarely test the behaviour they expect of
+                    the library, neither the library developer nor the library user will be aware of the new behaviour.
+                </p>
+
+                <p>
+                    As a library developer, you want to be able to see how a change in your library will affect your
+                    users before a new version of the library is deployed. More specifically, you want to gain insight
+                    into <strong>how users use the library</strong>, and want to see if and{" "}
+                    <strong>how changes affect users</strong>. This will allow you to determine whether the new version
+                    of the library is <strong>backwards compatible</strong>. Finally, after deploying the breaking
+                    changes, you want to notify the affected users of the changes and of a solution to the issue.
+                </p>
+            </div>
+
+            <div className="content-section bg-grey-light">
+                <h2>Schaapi's Approach</h2>
```

Comment:
The other titles are lowercase as well, so I think we should do "Schaapi's approach".